backend/src/app/sessionRoutes.ts

```diff
@@ -46,52 +50,53 @@ if (!sessionSigningSecret) {
 	process.exit(1);
 }
 
-const router = express.Router();
-
+const cookieName = process.env.COOKIE_SID;
```

Comment:
We need to add a new thing to .env? Shall we add something to .env.example too?

Reply:
nvm, I've seen that it gets added to the fargate instance, and it runs fine locally

Reply:
Ah yes, it falls back to the default cookie name, but it would probably be clearer to use our own specific cookie name, so I'll add it to the example env 👍